src/NLog/Layouts/JsonAttribute.cs

```diff
@@ -56,6 +56,18 @@ public JsonAttribute(string name, Layout layout)
         {
             this.Name = name;
             this.Layout = layout;
+            this.Encode = true;
```

Comment:
This is a bit inconsistent. The first ctor calls the third, but the second is fully separated. I would suggest that the second ctor also calls the third one (with `this(name, layout, true)